lib/Doctrine/ORM/Tools/SchemaValidator.php

```diff
@@ -258,14 +258,16 @@ public function validateClass(ClassMetadataInfo $class)
     /**
      * Checks if the Database Schema is in sync with the current metadata state.
      *
+     * @param bool $strictSync If TRUE, check that all tables in a database have a mapping files
+     *
      * @return bool
      */
-    public function schemaInSyncWithMetadata()
+    public function schemaInSyncWithMetadata($strictSync = false)
```

Comment:
Adding a new parameter to a non-final method is a breaking change. We cannot do that.

Reply:
Maybe you can do this in the next major release?